Jenkinsfile

```diff
@@ -313,6 +313,41 @@ def buildHipClangJobAndReboot(Map conf=[:]){
     }
 }
 
+def RunPerfTest(Map conf=[:]){
+    def dockerOpts="--device=/dev/kfd --device=/dev/dri --group-add video --group-add render --cap-add=SYS_PTRACE --security-opt seccomp=unconfined"
+    try {
+        (retimage, image) = getDockerImage(conf)
+        withDockerContainer(image: image, args: dockerOpts + ' -v=/var/jenkins/:/var/jenkins') {
+        timeout(time: 600, unit: 'MINUTES')
```

Comment:
[Q] 10 hours - isn't that too much?

Reply:
Probably, I dont know what the upper bound is, so I went with something that would clear it. What would be an acceptable time?

Reply:
As a rule of thumb, I would suggest 2X the normal time of the test (look at your branch at our CI; MI100/MI200 nodes seem the slowest ones)

Reply:
I heard that currently the test takes ~4 hours, so I can recommend `time: 480`

Reply:
I consider this as resolved.